InnerEye/ML/metrics.py

```diff
@@ -56,15 +57,15 @@ class InferenceMetricsForSegmentation(InferenceMetrics):
     """
     Stores metrics for segmentation models, per execution mode and epoch.
     """
-    data_split: ModelExecutionMode
+    execution_mode: ModelExecutionMode
     metrics: float
 
     def get_metrics_log_key(self) -> str:
         """
         Gets a string name for logging the metrics specific to the execution mode (train, val, test)
```

Comment:
comment outdated. Also, we probably want to break down by execution mode AND data split.

Reply:
Perhaps, but all I'm doing here is making our nomenclature consistent, i.e. when we say `data_splits` we mean a list of lists of patient ids, so when we say `data_split` we should mean a list of patient ids, not an execution mode. It made it hard for me to understand the code so I am just correcting that.